### PR TITLE
A Primitive Version of REPL

### PR DESCRIPTION
**Pull request by Cheny-chui**
<del>等大佬review @NeilKleistGao </del>
Related Issue:
- #3 
- #4

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+version = "3.5.3"
+runner.dialect = scala213
```
Comment (NeilKleistGao) [inline .scalafmt.conf]:
```suggestion
runner.dialect = scala213

```

File: src/main/scala/arcana/world/Main.scala [hunk 2]
```diff
@@ -2,7 +2,8 @@ package arcana.world
 
 object Main {
   def main(args: Array[String]): Unit = {
-    System.out.println("Hello World")
+    var version = "0.0";
```
Comment (NeilKleistGao) [inline src/main/scala/arcana/world/Main.scala]:
```suggestion
    val version = "0.0";
```
Do avoid using mutable variables.

File: src/main/scala/arcana/world/Repl.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package arcana.world
+
+import scala.io.StdIn
+
+object Repl {
```
Comment (NeilKleistGao) [inline src/main/scala/arcana/world/Repl.scala:5]: Why use `object` here?

File: src/main/scala/arcana/world/Repl.scala [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  val fooEvaluat = (input: String) => println(input)
+  val fooTypeEvaluat = (input: String) => println(input + ": type")
+  def evaluatArguments(args: Array[String]): Unit = {
+    var typeInfo = false
+    for (arg <- args) {
+      arg match {
+        // Add any argument here
+        case "--type" => typeInfo = true
+      }
+    }
```
Comment (NeilKleistGao) [inline src/main/scala/arcana/world/Repl.scala]:
```suggestion
    val needTypeInfo = args.foldLeft(false)((flag, arg) => flag || arg.equals("--type"))
```
Try to use `fold`, `reduce` and so on in the code.

**Review by NeilKleistGao — CHANGES_REQUESTED** [review]

**Comment by NeilKleistGao** [comment]
Please move the repl files into a new package named `arcana.death`. The current package `arcana.world` should only contain arguments parsing.

File: src/main/scala/arcana/world/Repl.scala (hunk 3 above)
Comment (Cheny-chui) [reply src/main/scala/arcana/world/Repl.scala:5]: there is only one Repl object in system

**Pull request closed without merging**